git/essentials/features-ii/debug-using-binary-search.md

```diff
@@ -13,6 +13,12 @@ type: normal
 
 category: must-know
 
+aspects:
+  
+  - introduction
+  - deep
```

Comment:
I would remove `introduction` and replace it with `workout`. This is very useful, but you will very rarely use this at a basic level.